odonto/templates/forms/demographics_form.html

```diff
@@ -0,0 +1,8 @@
+{% load forms %}
+{% input field="Demographics.nhs_number" label="NHS Number" %}
```

Comment:
FYI you shouldn't need 'NHS Number' its the verbose name of the field (I'm sure this is cut and pasted from opal but we should change it there too)